inference/core/roboflow_api.py

```diff
@@ -35,6 +35,7 @@
     "object-detection": "yolov5v2s",
     "instance-segmentation": "yolact",
     "classification": "vit",
+    "keypoint-detection": "yolov8n",
```

Comment:
most changes are `keypoints-detection` -> `keypoint-detection`, but this was a necessary addition to make sure model manager accepts type.

Reply:
👍